### PR TITLE
fix error agent view does not render correctly

### PR DESCRIPTION
**Pull request by matiasmoreno876**
Hi team, this resolve:

- https://github.com/wazuh/wazuh-kibana-app/issues/3229

image

### CODE REVIEW

File: public/controllers/agent/components/agents-preview.js [hunk 1]
```diff
@@ -120,15 +120,15 @@ export const AgentsPreview = compose(
...
+
   removeFilters(){
     this._isMount && this.setState({agentTableFilters: []})
   }
 
   render() {
     const colors = ['#017D73', '#bd271e', '#69707D'];
     return (
-      <EuiPage>
+      <EuiPage style={{ flexDirection: 'column'}}>
```
Comment (gabiwassan) [inline public/controllers/agent/components/agents-preview.js:131]: Could we put this in a CSS class?

File: public/controllers/agent/components/agents-preview.js [hunk 2]
```diff
@@ -236,7 +236,7 @@ export const AgentsPreview = compose(
                               <EuiToolTip
                                 position='top'
                                 content='View agent details'>
-                                  <a onClick={() => 
+                                  <a onClick={() =>
                                   this.props.tableProps.showAgent(
                                     this.lastAgent
                                   )}>{this.lastAgent.name}</a>
```
Comment (gabiwassan) [inline public/controllers/agent/components/agents-preview.js:242]: Can you separate this on a new arrow function method?

File: public/controllers/agent/components/agents-preview.js [hunk 3]
```diff
@@ -298,13 +298,13 @@ export const AgentsPreview = compose(
                     </div>
                     {this.props.resultState === 'loading' &&
                       (
-                      <div style={{ display: 'block', textAlign: "center", padding: 30}}>                        
+                      <div style={{ display: 'block', textAlign: "center", padding: 30}}>
```
Comment (gabiwassan) [inline public/controllers/agent/components/agents-preview.js:301]: This could be in a CSS too 🤓

**Review by gabiwassan — CHANGES_REQUESTED** [review]
Some legacy improvements 😃

**Review by pablomarga — APPROVED** [review]
Testing: :heavy_check_mark:

File: public/controllers/agent/components/agents-preview.js (hunk 2 above)
Comment (matiasmoreno876) [reply public/controllers/agent/components/agents-preview.js:242]: Yes, there I do, thanks for the observation +1

File: public/controllers/agent/components/agents-preview.js (hunk 1 above)
Comment (matiasmoreno876) [reply public/controllers/agent/components/agents-preview.js:131]: Yes, it is possible to make that change, thank you.

**Review by gabiwassan — APPROVED** [review]
LGTM! 🚀

**Pull request merged**